src/main/java/rs117/hd/overlays/ShaderOverlay.java

```diff
@@ -57,7 +57,7 @@
 
 @Slf4j
 @Singleton
-public class ShaderOverlay<T extends ShaderOverlay.Shader> extends Overlay {
+public class ShaderOverlay<T extends ShaderOverlay.Shader> extends HdOverlay {
```

Comment:
If we're making a `HdOverlay` class I think it makes sense to split `ShaderOverlay` into the actual shader-specific parts and anything non-shader into the new `HdOverlay` class. That way different `HdOverlays` can take advantage of the custom overlay logic in here, such as centering on the screen.